interfaces/mount/backend.go

```diff
@@ -58,7 +58,7 @@ func (b *Backend) Setup(snapInfo *snap.Info, confinement interfaces.ConfinementO
 	}
 	content := deriveContent(spec.(*Specification), snapInfo)
 	// synchronize the content with the filesystem
-	glob := fmt.Sprintf("%s.fstab", interfaces.SecurityTagGlob(snapName))
+	glob := fmt.Sprintf("snap.%s.*fstab", snapName)
```

Comment:
Yes, if you comment this out the test in this patch will fail.

Note that this is just a temporary change. I hope to transition away from security-tag based mount profiles as soon as possible. Then the glob can be simplified to `snap.*.fstab`